### DISCUSSION TITLE
Setting empty conditions list in sentry_alert_issue

### DISCUSSION THREAD

**Discussion by outdoortejan** [jianyuan/terraform-provider-sentry#365] [General]
What is the proper way to specify an empty list of `conditions` in the `sentry_issue_alert` resource? I'm actually trying to apply my config of an alert that already exists in the UI. I've tried every combination I can think of but the apply always either fails or is trying to add a new `conditions` configuration. What do I need to use in the conditions parameter if I want it to be an empty list as the sentry API returns?

**Comment by jianyuan** [comment]
Hi @ocdrums3, I am aware the documentation is not clear. The correct value should be an empty JSON array. e.g. `conditions = "[]"`.

**Reply by outdoortejan** [reply]
Thank you
